tests/Tests.Json.Decode.fs

```diff
@@ -1528,7 +1520,7 @@ Expecting a string but instead got: 12
                     Error(
                         """
 Error at: `$.user`
-Expecting an object at `user` but instead got:
+Expecting an object but instead got:
```

Comment:
Ok we "loose" an information in the second line message but because we output the error path it's ok I guess :)

Before we were probably giving twice the same information

Reply:
Yes, that was my impression too :)